src/main.rs

```diff
@@ -90,6 +95,7 @@ fn main() -> anyhow::Result<()> {
                 });
             }
         }
+        println!("{bold}Actual checks: {info}{}{reset}", actual_checks);
```

Comment:
```suggestion
        println!("{bold}Actual checks: {info}{actual_checks}{reset}");
```

I'd prefer to keep string interpolation inline if possible so it's more clear where `actual_checks` goes in the string.

Reply:
sure. would you like me to also do this for `println!("{bold}Estimated checks: {info}{}{reset}", estimated_checks);`?

Reply:
Yes please!